pip/nbconvert.file

```diff
@@ -1,2 +1,2 @@
-Requires: py2-nbformat py2-Jinja2 py2-Pygments py2-bleach py2-backports py2-defusedxml py2-entrypoints py2-mistune py2-pandocfilters py2-testpath
+Requires: py2-nbformat py2-Jinja2 py2-Pygments py2-bleach py2-backports py2-defusedxml py2-entrypoints py2-mistune py2-pandocfilters py2-testpath py3-nbformat
```

Comment:
better to create a new pip/py3-nbconvert.file file and add this py3 dependency there

Reply:
I was not aware of that option.. Done. Any chance we can clean up the existing files to follow this pattern consistently?